easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.5.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libgeotiff'
+version = '1.5.1'
+
+homepage = 'https://directory.fsf.org/wiki/Libgeotiff'
+description = """Library for reading and writing coordinate system information from/to GeoTIFF files"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['http://download.osgeo.org/geotiff/libgeotiff']
+sources = [SOURCE_TAR_GZ]
+checksums = ['f9e99733c170d11052f562bcd2c7cb4de53ed405f7acdde4f16195cd3ead612c']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('LibTIFF', '4.0.10'),
+    ('PROJ', '6.0.0'),
+    ('libjpeg-turbo', '2.0.2'),
+]
+
+configopts = ' --with-libtiff=$EBROOTLIBTIFF --with-proj=$EBROOTPROJ --with-zlib=$EBROOTZLIB'
```

Comment:
Sorry, I noticed this after initially thinking it looked good, but I think you have missed one of the config options:
```python
configopts += ' --with-jpeg=$EBROOTLIBJPEGMINTURBO'
```
I verified in my build log that I got `jpeg support......: no` without it, so it looks like it's still needed.